src/model/Position.java

```diff
@@ -1,42 +0,0 @@
-package model;
-
-public class Position {
-	public int x, y;
```

Comment:
consider x and y to by final and prefer to have then encapsulated, private with getter.